app/ydoc-shared/src/ast/mutableModule.ts

```diff
@@ -124,7 +135,7 @@ export class MutableModule implements Module {
     if (root) {
       root.syncToCode(code)
     } else {
-      this.replaceRoot(Ast.parse(code, this))
+      this.replaceRoot(parseModuleWithSpans(code, this).root)
```

Comment:
Could you replace docs stub for this function?

Reply:
`replaceRoot` is now `setRoot`, with docs; `parseModuleWithSpans` now has a `parseModule` helper for when spans are not wanted, and is documented.

Reply:
Sorry, I wasn't precise. I meant `syncToCode`. I could not attach a comment to function signature.

But all is ok, my goal was to reduce number of `TODO: Add docs` a bit.